CPAC/registration/registration.py

```diff
@@ -2806,12 +2805,13 @@ def coregistration(wf, cfg, strat_pool, pipe_num, opt=None):
         func_to_anat.inputs.inputspec.interp = cfg.registration_workflows[
         'functional_registration']['coregistration']['interpolation']
 
-        node, out = strat_pool.get_data('desc-reginput_bold')
+        node, out = strat_pool.get_data('sbref')
         wf.connect(node, out, func_to_anat, 'inputspec.func')
 
         if cfg.registration_workflows['functional_registration'][
             'coregistration']['reference'] == 'brain':
-            node, out = strat_pool.get_data('desc-brain_T1w')
+            # TODO: use JSON meta-data to confirm
```

Comment:
We should probably open an issue to try make sure this doesn't get lost in the shuffle

Reply:
Agreed. Though I won't be forgetting personally, as it's been stinging my mind for awhile now as part of my engine redesign finalization 😄